spec/schemas/_common.yaml

```diff
@@ -686,7 +686,10 @@ components:
             $ref: '#/components/schemas/NodeId'
     NodeId:
       type: string
-    Bytes:
+    StorageSize:
+      type: string
+      pattern: '\d+(\.\d+)?(b|kb|k|mb|m|gb|g|tb|t|pb|p)'
+    StorageType:
```

Comment:
This naming does not make sense to me. The name would imply this is types of storage (cold, warm, hdd, etc). When in actuality this is byte units.

Reply:
What would you prefer? My suggestion would be `BytesUnit` ?

Reply:
I think that's reasonable as we do have other `...Unit` enums.